pxemgr/pxemanager.go

```diff
@@ -386,8 +383,7 @@ func (mgr *pxeManagerT) getNextInternalIP() net.IP {
 		currentIP = incIP(currentIP)
 	}
 
-	panic(errors.New("unable to get a free ip"))
-	return net.IP{}
+	//panic(errors.New("unable to get a free ip"))
```

Comment:
I'm late here, but: for loop above `for ; ; ipLessThanOrEqual(currentIP, rangeEnd) {` is infinite. I think intention there was `for ipLessThanOrEqual(currentIP, rangeEnd)`. Then `panic` would be reachable.